app/src/main/res/values-nl/translation.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <!-- Language Names Here -->
+    <string-array name="languages">
+        <item>English</item>
+        <item>Dutch </item>
+    </string-array>
+
+    <!-- Edit your name here according to translated language -->
+    <string name="English_contributors">Vaibhav</string>
+    <string name="Dutch_contributors">Thijsi123</string>
```

Comment:
String `name` fields should start with lower case characters as per naming convention